zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/impl/AQLGenerator.scala

```diff
@@ -121,9 +135,9 @@ class AQLGenerator extends IQLGenerator {
         """.stripMargin
       )
       //TODO check if the vars are duplicated
-      val field: Field = ??? // get field from lookup table
+      val fieldMap: Map[String, Field] = schemaMap.flatMap(_._2.fieldMap) // get fields from lookup and query table
```

Comment:
then there is the problem, right? we removed one or more field(s) just because it has the same name with the first one?